### PR TITLE
Add ONNX export for owlv2

### PR DESCRIPTION
**Pull request by xenova**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests?

### CODE REVIEW

File: optimum/exporters/onnx/model_configs.py [hunk 1]
```diff
@@ -1085,6 +1085,10 @@ def outputs(self) -> Dict[str, Dict[int, str]]:
         return outputs
 
 
+class OwlV2OnnxConfig(OwlViTOnnxConfig):
+    DEFAULT_ONNX_OPSET = 14
```
Comment (fxmarty) [inline optimum/exporters/onnx/model_configs.py:1089]: Can you add a comment as to why this opset is needed?

**Review by fxmarty — APPROVED** [review]
LGTM

**Comment by fxmarty** [comment]
@xenova Can you also rebase/merge main?

**Comment by xenova** [comment]
Done! Turns out `opset=14` isn't necessary 😇

File: optimum/exporters/onnx/model_configs.py (hunk 1 above)
Comment (xenova) [reply optimum/exporters/onnx/model_configs.py]: Removed since it's not actually needed.

**Comment by fxmarty** [comment]
@xenova Could you either push an empty commit or merge main (there was an issue with pytest that is now pinned)?

Can you also confirm that the exported model behaves as expected?

**Comment by fxmarty** [comment]
bump

**Comment by xenova** [comment]
Done! (sorry for the delay; missed previous notification)

**Review by fxmarty — APPROVED** [review]

**Pull request merged**